### ISSUE TITLE
restart needed after updating to 1.3-Preview 1/Experimental 1

### ISSUE THREAD

**Issue by ghost1372** [microsoft/WindowsAppSDK#3554]
### Describe the bug

after updating project to wasdk 1.3-preview1 or experimental1 we need to restart visual studio. if not, we have a compile issues

```
1>  C:\Users\...\.nuget\packages\microsoft.windowsappsdk\1.3.230202101-experimental1\buildTransitive\Microsoft.UI.Xaml.Markup.Compiler.interop.targets(523,17): error MSB4064: The "PrecompiledHeaderFile" parameter is not supported by the "CompileXaml" task loaded from assembly: Microsoft.UI.Xaml.Markup.Compiler, Version=1.0.0.0, Culture=neutral, PublicKeyToken=de31ebe4ad15742b from the path: C:\Users\...\.nuget\packages\microsoft.windowsappsdk\1.2.230118.102\tools\net472\Microsoft.UI.Xaml.Markup.Compiler.dll. Verify that the parameter exists on the task, the <UsingTask> points to the correct assembly, and it is a settable public instance property.
1>  C:\Users\...\.nuget\packages\microsoft.windowsappsdk\1.3.230202101-experimental1\buildTransitive\Microsoft.UI.Xaml.Markup.Compiler.interop.targets(491,9): error MSB4063: The "CompileXaml" task could not be initialized with its input parameters.
```

### Steps to reproduce the bug

1.update current project to wasdk 1.3-preview or exp

### NuGet package version

Windows App SDK 1.3 Preview 1: 1.3.230228005-preview1

### Packaging type

Packaged (MSIX), Unpackaged

### Windows version

Windows 11 version 22H2 (22621, 2022 Update)

### IDE

Visual Studio 2022

**Comment by MSLukeWest** [comment]
I still see this issue in the 1.3.230331000 release.  Restarting VS still fixes the issue.

**Comment by mattleibow** [comment]
This does not really fix it for cases where we have to multi-targer a project and end up with both versions.

I think the solution is to have different dll versions for each build task assembly. I see they are already strong named, so the final step is to have different version numbers.

**Comment by mattleibow** [comment]
Added an internal tracking issue: https://dev.azure.com/microsoft/OS/_workitems/edit/44263619

**Comment by dotMorten** [comment]
Restarting does _NOT_ fix the issue if you have multiple versions in your solution. It's more or less a blocker for me.
Example: Class library that works with any version 1.x and up.
App that uses said class library but uses the latest WinAppSDK.

It especially falls apart when you're building MAUI applications but want to use a newer version of WinAppSDK for the app but keep the class lib compatible with all versions.

I was surprised to see this wasn't resolved in 1.3.1 - it should be as simple as just ensure the build task has a new identify so Visual Studio can tell the two assemblies apart.

**Comment by bpulliam** [comment]
Fixed in 1.3.2
